patches/011-122-fixes.patch

```diff
@@ -0,0 +1,71 @@
+diff --git a/src/crypto/tls/boring_test.go b/src/crypto/tls/boring_test.go
+index 3f88fbb3b8..dbd6e6600c 100644
+--- a/src/crypto/tls/boring_test.go
++++ b/src/crypto/tls/boring_test.go
+@@ -75,7 +75,10 @@ func isBoringCipherSuite(id uint16) bool {
+ 		TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,
+ 		TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,
+ 		TLS_RSA_WITH_AES_128_GCM_SHA256,
+-		TLS_RSA_WITH_AES_256_GCM_SHA384:
++		TLS_RSA_WITH_AES_256_GCM_SHA384,
```

Comment:
@ueno also wanted to call your attention to this change to ensure cipher suite is correct.

Reply:
Afaik TLS_CHACHA20_POLY1305_SHA256 is still not approved in FIPS mode.